### PR TITLE
[Cleanup] Migrating view action tests

### PR DESCRIPTION
**Pull request by chadhietala**

### CODE REVIEW

File: packages/ember-views/tests/views/view/actions_test.js [hunk 1]
```diff
@@ -1,102 +0,0 @@
...
-    actions: {
-      poke() {
-        ok(true, 'poked');
-      }
-    }
-  }).create();
-  view.send('poke');
-});
-
-QUnit.test('A handled action can be bubbled to the target for continued processing', function() {
```
Comment (chadhietala) [inline packages/ember-views/tests/views/view/actions_test.js]: Removed this.

**Comment by chadhietala** [comment]
Maybe some of this will be covered by #13491?

File: packages/ember-views/tests/views/view/actions_test.js (hunk 1 above)
Comment (chadhietala) [inline packages/ember-views/tests/views/view/actions_test.js]: Although I'm think we need to be sending an action through the boundary.

**Comment by Joelkang** [comment]
Oops, @chadhietala, I didn't know you had done this. I moved all the action-related stuff with this PR https://github.com/emberjs/ember.js/pull/13532

**Comment by locks** [comment]
I'm closing this issue as this migration seems to have been addressed somewhere else.
Please correct me if I misunderstood something!

**Pull request closed without merging**